apps/pipelines/nodes/nodes.py

```diff
@@ -233,7 +243,7 @@ class Passthrough(PipelineNode):
 
     def _process(self, input, state: PipelineState, node_id: str) -> str:
```

Comment:
nit: you should change the output type to `PipelineState` here.

Reply:
https://github.com/dimagi/open-chat-studio/pull/869/commits/d166b39ff88e83eedd3aa7f38909bf98417781e6